lib/matplotlib/__init__.py

```diff
@@ -1553,7 +1553,12 @@ def _init_tests():
         warnings.warn(
             "matplotlib is not built with the correct FreeType version to run "
             "tests.  Set local_freetype=True in setup.cfg and rebuild. "
-            "Expect many image comparison failures below.")
+            "Expect many image comparison failures below. "
+            "Expected {0} != found {1}".format(
+                ft2font.__freetype_version__,
+                LOCAL_FREETYPE_VERSION
```

Comment:
I am not sure if `setup.py clean` is sufficient to remove everything that needs to be, I tend to use `git clean -xfd` to reset things :wink:.

There is also the `MPLLOCALFREETYPE` env which if it is set to anything true will use the local freetype (this is what I do).